### PR TITLE
TX-8 Scout Rifle impact ammo rebalance

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
Per title.

Stats have been changed as follows;
- Penetration reduced from 45 > 30. Seriously.
- Stagger and knockback removed.
- Shots will now pierce through targets.

## Why It's Good For The Game
TX-8 impact ammo has been really dumb for too long. Stagger, slowdown, and knockback on a gun with 25 damage, 45 penetration, and 5 sunder per bullet.

## Changelog
:cl: Lewdcifer
balance: TX-8 impact ammo; 45 > 30 pen, stagger and knockback removed, shots pierce through targets.
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
```
MJP does not give a shit despite me nagging about this for ages
```
I didn't even realize the gun he was shooting you with if you're angry at me, and I said owned because I just saw someone die in a 1x1 closet, didn't really mean it offensively.
Other than that that's like, the only time of me being able to remember you complaining about this, and looking at your discord history that seems to line up.

Other than that, this sort of removes High Impacts entire gimmick of hilariously high pen and slowdown, but personally I'd drop it to 30 or 35 pen, and add penetration even if it is quite situational. I'd keep the knockback because it let's things get away decently easier, and keep the slowdown. That'd give it something over incend.

**Review by BraveMole — APPROVED** [review]
mjp said it was fine

**Comment by EvilMelonTheSecond** [comment]
Do piercing bullets even have a niche? I'm having a hard time visualizing a scenario where 2 xeno line up to get shot by an automatic rifle. In the best cases you will hit an extra xeno 1-2 times per mag, not worth the trade at all in my opinion

**Review by BraveMole — APPROVED** [review]

**Comment by Lumipharon** [comment]
As above, there is zero reason to actually buy this ammo with this change. Unless you're like, shooting a boiler behind a defender, I guess?

**Comment by KrabSpider** [comment]
Maybe inverse the sunder on regular TX8 bullets and the impact mags and raise the damage to 30?

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -573,13 +573,13 @@ datum/ammo/bullet/revolver/tp44
...
-	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING|AMMO_PASS_THROUGH_MOVABLE
 	damage = 25
-	penetration = 45
+	penetration = 30
 	sundering = 5
 
-/datum/ammo/bullet/rifle/tx8/impact/on_hit_mob(mob/M, obj/projectile/P)
-	staggerstun(M, P, max_range = 20, stagger = 2, slowdown = 1, knockback = 1, shake = 0)
+/datum/ammo/bullet/rifle/tx8//on_hit_mob(mob/M, obj/projectile/P)
```
Comment (hyper2snyper) [inline code/modules/projectiles/ammo_datums.dm:581]:
```suggestion
/datum/ammo/bullet/rifle/tx8/on_hit_mob(mob/M, obj/projectile/P)
```

**Review by hyper2snyper — CHANGES_REQUESTED** [review]

**Pull request merged**